nss-cohort-36/bangazon-api-abracadabras_remix
# Jb model refactor

Steps to Test:
1. Checkout Master
2. Git fetch --all
3. Git Checkout jb-modelRefactor
4. Run Virtual Environment via Terminal Command: source./bangazonApiEnv/Scripts(or the mac Command Equivalent)/activate
5. Run Terminal Command: python manage.py runserver
6. Then Open Browser and Enter: localhost:8000
7. Test Endpoints /producttype and then /producttype/1 or equivalent
8. If no content shows check your individual database to ensure you have data in appropriate tables.
